app/views/AttachmentView.js

```diff
@@ -8,7 +8,7 @@ import RNFetchBlob from 'rn-fetch-blob';
 import { Video } from 'expo-av';
 import SHA256 from 'js-sha256';
 import { withSafeAreaInsets } from 'react-native-safe-area-context';
-
+import * as FileSystem from 'expo-file-system';
```

Comment:
Why not `rn-fetch-blob`?

Reply:
`expo-FileSystem` had a more intuitive API and we were already using that so thought using this might be easier.